File: AtCoder/abc207/A.swift
private func readLn() -> String {
	return readLine()!
}

private func readLns() -> [String] {
	return readLn().split(separator: " ").map{ String($0) }
}

private func readInts() -> [Int] {
	return readLns().compactMap{ Int($0) }
}

var arr = readInts()
arr.sort()
print(arr[1] + arr[2])

File: AtCoder/abc207/A.kt
private fun readLn() = readLine()!!
private fun readLns() = readLn().split(" ")
private fun readInts() = readLns().map { it.toInt() }

fun main() {
    var arr = readInts()
    arr = arr.sorted()
    print(arr[1] + arr[2])
}